api/cf-rpc-types/src/broker.rs

```diff
@@ -73,6 +73,7 @@ pub type TransactionInIdFor<C> = <<C as Chain>::ChainCrypto as ChainCrypto>::Tra
 #[derive(Serialize, Deserialize)]
 pub enum TransactionInId {
 	Bitcoin(TransactionInIdFor<cf_chains::Bitcoin>),
+	Ethereum(TransactionInIdFor<cf_chains::Ethereum>),
 	// other variants reserved for other chains.
```

Comment:
Should this include Arbitrum?

Reply:
Yeah... I think this must have got lost in the cherry pick because the file location changed... will fix.